### PR TITLE
Editing index with my special touch

### PR DESCRIPTION
**Pull request by sid-s1**

### CODE REVIEW

**Review by Chrispy1987 — APPROVED** [review]
It is I...

File: index.html [hunk 1]
```diff
@@ -4,9 +4,12 @@
...
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>DIS OUR PROJ</title>
 </head>
 <body>
-    
+    <h1>Who goes here?</h1>
+    <button>
+        your SOUL
```
Comment (davebdev) [inline index.html:12]: Maybe capitalise the 'Your'?

**Review by davebdev — CHANGES_REQUESTED** [review]
Check the comments

File: index.html (hunk 1 above)
Comment (sid-s1) [reply index.html:12]: How about no?

**Review by davebdev — APPROVED** [review]

**Pull request merged**